pkg/internal/wordpress/pod_template.go

```diff
@@ -350,6 +351,48 @@ func (wp *Wordpress) initContainers() []corev1.Container {
 	return containers
 }
 
+func (wp *Wordpress) readinessProbe() *corev1.Probe {
+	// If the HTTPGetAction doesn't have any Host parameter it will use pod's IP address as Host.
+	// This is helpful because Wordpress may not be installed and in this case it will redirect to
+	// Spec.Domains[0]. If the same host was used in the initial request as Spec.Domains[0], k8s will follow
+	// the Location header, which may point to an unreachable address, thus making the pod UnHealthy.
+	//
+	// Refs:
+	//		* https://github.com/kubernetes/kubernetes/pull/75416
+	//		* https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/
+	//				Any code greater than or equal to 200 and less than 400 indicates success.
```

Comment:
I think it looks better if you are using 2 (or 4) spaces indentation.
```suggestion
	//  * https://github.com/kubernetes/kubernetes/pull/75416
	//  * https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/
	//    Any code greater than or equal to 200 and less than 400 indicates success.
```